extensions/cache/runtime/src/main/java/io/quarkus/cache/build/CacheBuildRecorder.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.cache.build;
```

Comment:
Same remark here about the `runtime` sub package.

You should have:
- `io.quarkus.cache`: the API
- `io.quarkus.cache.runtime`: everything else (obviously you can use subpackages)